apps/workers/email-pre-renderer/src/emailPreRender.ts

```diff
@@ -0,0 +1,57 @@
+import { Subscribers, getSupabaseClient } from 'db'
+import { RedisClientType, createClient as createRedisClient } from 'redis'
+import { Entities, connectToQueue } from 'shared'
+import { RedisPrefix } from 'shared/src/enums/redis'
+import { getAllPaginated } from 'shared/src/services/getAllPaginated'
+import { CONFIG } from './config'
+import { renderFooter } from './renderFooter'
+import { renderHeader } from './renderHeader'
+import { sendToQueue } from './sendToQueue'
+
+export async function emailPreRender() {
+  const redisClient = createRedisClient() as RedisClientType
+  const channel = await connectToQueue({
+    password: CONFIG.RABBITMQ_PASS,
+    user: CONFIG.RABBITMQ_USER,
+  })
+  const supabase = getSupabaseClient()
+  await redisClient.connect()
```

Comment:
Please keep related code together.

Move this `await redisClient.connect()` closer to the redisClient declaration.